src/target/stm32f4.c

```diff
@@ -355,3 +370,30 @@ static bool stm32f4_cmd_option(target *t, int argc, char *argv[])
 	}
 	return true;
 }
+
+static bool stm32f4_cmd_psize(target *t, int argc, char *argv[])
+{
```

Comment:
Please put braces on the same line as the if and else, and add braces to the ifs below (strcmp's)